app/views/notify_user/note_added.html.haml

```diff
@@ -0,0 +1,6 @@
+%h2 A note was added.
+
+%p
+  A note was added for the following client. 
+%p
+  = link_to "Client ##{@client.id}", client_notes_url(@client)
```

Comment:
This will need to link to the window version of notes, since most people don't have access to the client section.

Reply:
Same as files, this should check user access to `:can_edit_client_notes` and `can_edit_window_client_notes`.  No link if you don't have either of those, appropriate link if you only have window and not client access.